system variable: rename tidb_ddl_distribute_reorg to tidb_enable_dist_task (#13298)
pingcap/docs: system-variables.md

</CustomContent>

### tidb_enable_dist_task <span class="version-mark">New in v7.1.0</span>

> **Warning:**
>
> - This feature is still in the experimental stage. It is not recommended to enable this feature in production environments.

- Scope: GLOBAL
- Persists to cluster: Yes
- Default value: `OFF`

> - This variable is used to control whether to enable [the distributed execution framework](/tidb-distributed-execution-framework.md). After enabling distributed execution, backend tasks such as DDL and import will be jointly completed by multiple TiDB nodes in the cluster.

Reviewer: As tidb-distributed-execution-framework.md will be ready next month, the reference to it can be removed for now.

> - This variable is used to control whether to enable [the distributed execution framework](/tidb-distributed-execution-framework.md). After enabling distributed execution, backend tasks such as DDL and import will be jointly completed by multiple TiDB nodes in the cluster.

Reviewer: ```suggestion
- This variable is used to control whether to enable the distributed execution framework. After it is enabled, backend tasks such as DDL and import will be jointly completed by multiple TiDB nodes in the cluster.
```

> - This variable is used to control whether to enable [the distributed execution framework](/tidb-distributed-execution-framework.md). After enabling distributed execution, DDL, Import and other supported backend tasks will be jointly completed by multiple TiDB nodes in the cluster.

Author: ```suggestion
- This variable is used to control whether to enable [the distributed execution framework](/tidb-distributed-execution-framework.md). After enabling distributed execution, backend tasks such as DDL and import will be jointly completed by multiple TiDB nodes in the cluster.
```

> - In TiDB v7.1.0, the framework supports distributedly executing the `ADD INDEX` statement for partitioned tables.

Author: ```suggestion
- In TiDB v7.1.0, the framework supports distributedly executing only the `ADD INDEX` statement for partitioned tables.
```
